assets/src/dashboard/app/views/editorSettings/adManagement/test/adManagement.js

```diff
@@ -29,6 +29,11 @@ import AdManagement, { TEXT } from '..';
 describe('Editor Settings: Ad Management group settings <AdManagement />', function () {
   let adNetwork;
   let mockUpdate;
+  const defaultsiteKitStatus = {
```

Comment:
```suggestion
  const defaultSiteKitStatus = {
```

Reply:
Copy and paste from https://github.com/google/web-stories-wp/blob/f459efbc4a2c9546a0ca2fed1301e4f222fa2340/assets/src/dashboard/app/views/editorSettings/googleAnalytics/test/googleAnalytics.js#L30

Reply:
Let's fix it there too 👍

Reply:
Fixed in https://github.com/google/web-stories-wp/pull/7622/commits/93131039b2becc54fe34cbb15b6824c9e349d55e